src/control/SConscript

```diff
@@ -136,6 +136,9 @@ def scons():
     denv.AppendENVPath('GOPATH', denv['GOPATH'])
     denv['ENV']['GOBIN'] = join(gopath, "bin")
 
+    prereqs.require(denv, 'cart')
+    denv.AppendENVPath("CGO_CFLAGS", denv.subst("-I$CART_PREFIX/include"))
+
```

Comment:
I think the build failure you're getting is because you forgot to adjust the other CGO_CFLAGS stuff below, c.f. https://github.com/daos-stack/daos/pull/1579/files#diff-86600e982223290eae2fe2b93d370de7R174-R178

You have to add the `sep=" "` param otherwise it gloms the new stuff onto the existing value with ":".